assets/src/edit-story/app/prepublish/warning/test/accessibility.js

```diff
@@ -234,14 +235,15 @@ describe('Pre-publish checklist - accessibility issues (warnings)', () => {
           alt: '',
         },
       };
-      expect(
-        accessibilityChecks.videoElementMissingDescription(element)
-      ).toStrictEqual({
-        message: MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.MAIN_TEXT,
-        help: MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.HELPER_TEXT,
-        elementId: element.id,
-        type: 'warning',
-      });
+      const test = accessibilityChecks.videoElementMissingDescription(element);
+      expect(test.message).toStrictEqual(
+        MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.MAIN_TEXT
+      );
+      expect(test.help).toStrictEqual(
+        MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.HELPER_TEXT
+      );
+      expect(test.elementId).toStrictEqual(element.id);
+      expect(test.type).toStrictEqual('warning');
```

Comment:
Just for my own curiosity: Was this done because `toStrictEqual` fails if there are more properties?

Reply:
Yes

Reply:
💭 This works great! Not important but if we don't want too much boiler plate code could we use `expect.objectContaining`? I don't think we need to change this though
```suggestion
      const test = accessibilityChecks.videoElementMissingDescription(element);
      expect(test).toStrictEqual(expect.objectContaining({
          message: MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.MAIN_TEXT,
          help: MESSAGES.ACCESSIBILITY.MISSING_VIDEO_DESCRIPTION.HELPER_TEXT,
          elementId: element.id,
          type: 'warning'
      }));
```